archinstall/lib/models/mirrors.py

```diff
@@ -147,9 +149,9 @@ def __eq__(self, other: object) -> bool:
 
 
 class SignCheck(Enum):
-	Never = 'Never'
-	Optional = 'Optional'
-	Required = 'Required'
+	Never = tr('Never')
```

Comment:
I'm not sure we should translate this, it's the actual value stored in the mirror configuration